django/contrib/admin/static/admin/css/forms.css

```diff
@@ -142,7 +142,7 @@ form .aligned select + div.help {
     padding-left: 10px;
 }
 
-form .aligned ul li {
+form .aligned div li {
```

Comment:
No **.aligned ul**  is renamed with a div tag because of ticket #32338 [https://code.djangoproject.com/ticket/32338](url) which had accessibility issues with Django forms RadioSelect and CheckBoxMultiple too. You can see the changes in the HTML file here [https://github.com/django/django/commit/5942ab5eb165ee2e759174e297148a40dd855920](url)